pkg/postgresql/utils.go

```diff
@@ -362,3 +363,25 @@ func GetTimelinesHistory(ctx context.Context, timeline uint64, replConnParams Co
 func IsValidReplSlotName(name string) bool {
 	return ValidReplSlotName.MatchString(name)
 }
+
+// isStreaming returns error if the PostgreSQL is not streaming WALs from master
```

Comment:
this should be phrased in a positive way first - i.e. what the purpose is and then describe the error conditions (if any):
```go
// isStreaming determines whether the PostgreSQL is streaming WALs from (the,a?) master.
// Returns NotFound if no receiver configured for WALs.
```
Please correct the comment if it does not reflect the full semantics.